wls-broadcast-service/src/main/resources/application.yml

```
@@ -2,6 +2,10 @@ spring:
   application.name: @project.artifactId@
   banner.location: banner.txt
 
+service:
+    info:
+        oid: WLS-BROADCAST
```

Einrückung weicht ab

```suggestion
service:
  info:
    oid: WLS-BROADCAST
```